src/pages/product/productShow/ProductShowPage.ts

```diff
@@ -22,6 +25,14 @@ class ProductShowPage extends BasePageModel {
   get recordStockButton() {
     return this.page.getByRole('link', { name: 'Record stock' });
   }
+
+  get inStockTab() {
+    return this.page.getByRole('link', { name: 'In Stock' });
+  }
+
+  get stockHistoryTab() {
+    return this.page.getByRole('link', { name: 'Stock History' });
+  }
```

Comment:
You can create a function that takes the tab name as a parameter
```suggestion
const openTab = (tabName) => {
   return this.page.getByRole('link', { name: tabName });
}
```

Reply:
Yes, I could but we are in general following approach and separating all tabs and elements on each page so I just continue following this approach as it has been already done in other tests.